unity-client/Assets/ABConverter/Wrappers/Implementations/Default/WebRequest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections;
+using System.Net.Http;
+using DCL.Helpers;
+using Unity.EditorCoroutines.Editor;
+using UnityEngine.Networking;
+using UnityGLTF;
+
+namespace DCL
+{
+    public static partial class UnityEditorWrappers
+    {
+        public class WebRequest : IWebRequest
+        {
+            private static int ASSET_REQUEST_RETRY_COUNT = 5;
+
+            public void GetAsync(string url, System.Action<DownloadHandler> OnCompleted, System.Action<string> OnFail)
+            {
+                EditorCoroutineUtility.StartCoroutine(GetAsyncCoroutine(url, OnCompleted, OnFail), this);
+            }
+
+            IEnumerator GetAsyncCoroutine(string url, System.Action<DownloadHandler> OnCompleted, System.Action<string> OnFail)
+            {
+                UnityWebRequest req;
+
+                int retryCount = ASSET_REQUEST_RETRY_COUNT;
+
+                do
+                {
+                    req = UnityWebRequest.Get(url);
+                    req.SendWebRequest();
```

Comment:
I think it's easier to just yield here instead of actively waiting for `req.IsDone`